orchestrator/cli/domain_gen_helpers/fixed_input_helpers.py

```diff
@@ -0,0 +1,142 @@
+from typing import Dict, List, Set, Union
+
+from more_itertools import flatten
+from sqlalchemy.orm import Query
+from sqlalchemy.sql.expression import Delete, Insert, Update
+
+from orchestrator.cli.domain_gen_helpers.helpers import get_user_input, sql_compile
+from orchestrator.cli.domain_gen_helpers.product_helpers import get_product_id, get_product_ids
+from orchestrator.db.models import FixedInputTable
+
+
+def map_update_fixed_inputs(product_diffs: Dict[str, Dict[str, Set[str]]]) -> Dict[str, Dict[str, str]]:
+    """Map fixed inputs to update.
+
+    Args:
+        - product_diffs: Dict with product differences.
+            - key: product name
+            - value: Dict with differences between model and database.
+                - key: difference name, 'missing_fixed_inputs_in_model' and 'missing_fixed_inputs_in_db' are used to check if a fixed input can be renamed.
+                - value: Set of fixed input names.
+
+    Returns: Dict with updated fixed inputs mapped by product.
+        - key: product name.
+        - value: Dict with updated fixed inputs.
+            - key: old fixed input name.
+            - value: new fixed input name.
+    """
+    print("--- UPDATE FIXED INPUT DECISIONS ---")  # noqa: T001, T201
+
+    def should_rename(product_name: str, product_diff: Dict[str, Set[str]]) -> Dict[str, str]:
+        db_props = list(product_diff.get("missing_fixed_inputs_in_model", []))
+        model_props = list(product_diff.get("missing_fixed_inputs_in_db", []))
+
+        if len(db_props) == 1 and len(model_props) == 1:
+            should_rename = get_user_input(
+                f"rename fixed input {db_props} to {model_props} for product ['{product_name}'] (y/N): ",
+                "n",
+            )
+            if should_rename == "y":
+                product_diffs[product_name]["missing_fixed_inputs_in_model"] = set()
+                product_diffs[product_name]["missing_fixed_inputs_in_db"] = set()
+                return {db_props[0]: model_props[0]}
+        return {}
+
+    return {name: should_rename(name, diff) for name, diff in product_diffs.items()}
+
+
+def generate_create_fixed_inputs_sql(
+    fixed_inputs: Dict[str, Set[str]], inputs: Dict[str, Dict[str, str]], revert: bool = False
+) -> List[str]:
+    """Generate sql strings to create product blocks.
```

Comment:
```suggestion
    """Generate sql strings to create fixed inputs.
```

Reply:
Couldn't batch update the docstrings, so did it manually.